example_with_tags_lang_test.go

```diff
@@ -13,6 +13,7 @@ func Example_withTagsLang() {
 		StringENG string `faker:"lang=eng"`
 		StringCHI string `faker:"lang=chi"`
 		StringRUS string `faker:"lang=rus"`
+		StringJA string `faker:"lang=ja"`
```

Comment:
Isn't the Japanese code `jpn`?
Reference: [here](https://www.loc.gov/standards/iso639-2/php/code_list.php)
```suggestion
		StringJPN string `faker:"lang=jpn"`
```

Reply:
Thanks for pointing that out.
I fixed it.